code/modules/power/singularity/emitter.dm

```diff
@@ -124,6 +124,8 @@
 				to_chat(user, "<span class='notice'>You turn on \the [src].</span>")
 				src.shot_number = 0
 				src.fire_delay = maximum_fire_delay
+				message_admins("Emitter turned on by [ADMIN_LOOKUPFLW(user)] in [ADMIN_COORDJMP(src)]",0,1)

```

Comment:
message_admins only has 1 parameter

Reply:
What the actual fuck, how was this never fixed in 6 years